eventloop.go

```diff
@@ -249,7 +249,9 @@ func (stw *Wallpaper) EventLoop(verbose bool, setWallpaperFunc func(string) erro
 		fmt.Println("Using the Simple Timed Wallpaper format.")
 	}
 
-	stw.SetInitialWallpaper(verbose, setWallpaperFunc)
+	if err := stw.SetInitialWallpaper(verbose, setWallpaperFunc); err != nil {
+		fmt.Print(err)
```

Comment:
Nice find! What do you think about returning the error here, instead of printing it?

Reply:
Well, depends on what you want to achieve. I have an impression, that currently if one image is missing program will run anyway, just ignore this one image. If it returns here, program can work/not work depending on hour you run it. Unless all errors always end program - then you will not allow missing files. 

Not sure what is better.